selftests/unit/test_loader.py

```diff
@@ -50,6 +50,45 @@ def test(self):
     main()
 """
 
+AVOCADO_TEST_TAGS = """#!/usr/bin/env python
+from avocado import Test
+from avocado import main
+
+import time
+
+class DisabledTest(Test):
```

Comment:
The point here is to test the feature, not to advertise it.  We've talked about before how Avocado's own tests should (or not) use snippets or refer to external files, and we've settled that it's a good idea to use snippets.

I'm planning a nice set of example tests and documentation on v1.